### DISCUSSION TITLE
Upload Proof page doesnt giving me any feedbacks i dont know what to do

### DISCUSSION THREAD

**Discussion by berk-efe** [community/community#67396] [GitHub Education]
### Hi everyone,

Hi i am a student, i did everything but at the last part i took a photo of my student document.
Also i tried to upload it too but it didnt work because it told me that its so small then i resized it on Adobe Illustrator than it told me that its coming from a third party app.
I take the document from Turkish goverment official website but it only gives pdf and 700x600 document size.(its not possible to take bigger)
i tried everything but it kept giving me errors. after a while it stopped giving me errors and just does nothing.
i guess it broke i dont know what to do

**Comment by MaryaBaloch20** [comment]
GitHub Education typically accepts image uploads in JPEG, PNG, or GIF formats. Ensure that your resized image is saved in one of these formats.
Or, If the image upload continues to cause issues, consider converting your image to a PDF format. You mentioned that you obtained the document as a PDF from the Turkish government website. You can try uploading the PDF instead of the image.

If none of the above solutions work and you suspect there may be an issue with the GitHub Education platform, consider reaching out to GitHub Support for assistance. They can provide guidance and help you resolve any technical problems.
